### PR TITLE
landing page

### PR DESCRIPTION
**Pull request by monicakogler**
Screen Shot 2020-09-29 at 4 37 10 PM

## landing page
- adds landing page w/ list of geographies
- coming soon for cities w/o guides yet

## guide page
- puts LA guide on its own URL
- adds link to index
- adds hover states to names
- removes footer

## misc
- compressed logos
- noticed that peter's link stuff from https://github.com/tvler/retreat-left/pull/4 is gone? intentional? easy to add back?

### CODE REVIEW

File: pages/la-nov-20.tsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                className="pr2"
+              >
+                <path d="M6 9L2 5L6 1" className="caret" strokeWidth="2" />
+              </svg>
+              All cities
+            </a>
+          </Link>
+
+          <div className="relative">
+            <img alt="DSA LA logo" src="/dsa-la@2x.png" className="h3 w-auto" />
```
Comment (monicakogler) [inline pages/la-nov-20.tsx:30]: @tvler u cool with sizing logos like this? then they can all be sized the same way. yuri trained me to make things the same height 🙎‍♂️

Comment (tvler) [reply pages/la-nov-20.tsx:30]: yea! good idea

**Comment by tvler** [comment]
> noticed that peter's link stuff from #4 is gone? intentional? easy to add back?

Yea I removed the link anchoring stuff in this pull https://github.com/tvler/retreat-left/pull/9 unfortunately :(

Here's what I wrote up about it:

> Also, removes the anchoring onClick now, as it could potentially cause people to not see the top of the page if they're sent a url on iOS which tries to hide URL params (this happened to eric accidentally & it would suck if someone accidentally tweets this out with an anchor tag)

**Comment by monicakogler** [comment]
ah ok right, i was too excited by the hover state mobile fix at the time to make that connection...

File: pages/index.tsx [hunk 2]
```diff
@@ -1,161 +1,78 @@
...
-            className="cool-red fw6 no-underline"
+            className="cool-red fw6 no-underline underline-pointer-hover"
             href="https://github.com/tvler/retreat-left"
           >
             GitHub
           </a>
           . Want to publish a voter guide for your county?{" "}
           <a
-            className="cool-red fw6 no-underline"
+            className="cool-red fw6 no-underline underlinepointer-hover"
```
Comment (tvler) [inline pages/index.tsx:75]: this should be underline-pointer-hover

**Review by tvler — APPROVED** [review]
it's literally perfect

**Comment by tvler** [comment]
❤️

File: pages/la-nov-20.tsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
+import { Fragment } from "react";
+import { NextPage } from "next";
+import Link from "next/link";
+
+import { data } from "../data";
+
+const LA: NextPage = () => {
```
Comment (tvler) [inline pages/la-nov-20.tsx:7]: Cute lol

**Pull request merged**